tilequeue/command.py

```diff
@@ -361,6 +361,11 @@ def parse_layer_data(query_cfg, template_path, reload_templates, cfg_path):
     layer_data = []
     all_layer_data = []
     post_process_data = []
+    area_threshold = query_cfg.get('area-inclusion-threshold')
+    try:
+        area_threshold = int(area_threshold)
+    except (ValueError, TypeError):
+        area_threshold = 1
```

Comment:
Do we want to hide the error here, or do something more like:

```python
area_threshold = int(query_cfg.get('area-inclusion-threshold', 1))
```

And allow a config error to stop the process?